JitsiMediaDevices.js

```diff
@@ -138,6 +138,18 @@ const JitsiMediaDevices = {
      */
     emitEvent(event, ...args) {
         eventEmitter.emit(event, ...args);
+    },
+
+    /**
+     * Returns whether or not the current browser is limited to supporting only
+     * audio send and receive.
+     *
+     * @returns {boolean}
+     */
+    supportsOnlyAudio() {
```

Comment:
how about inverting this and doing `supportsVideo`

Reply:
supportVideoOnly or separate checks for supporting audio and supporting video?

Reply:
yeah, could make it totally dynamic and have both